src/Dev/DevelopmentAdmin.php

```diff
@@ -190,6 +176,33 @@ protected static function get_links()
         return $links;
     }
 
+    protected function getLinks(): array
```

Comment:
If this is the replacement for `get_links`, please deprecate the old method - but in any case, why not just put this logic in the existing method?

Reply:
`get_links()` is a static method and thus can't call `$this->canViewAll()` - which needs to access `$this->getRequest()` so can't be static itself